scripts/update-contributions.js

```diff
@@ -44,4 +46,7 @@ Group.findAll({
       .catch(err => {
         console.log(`WARNING: ${err.message}`);
```

Comment:
Well, it *can*, but should it be?

Given the following:

```js
const baz = () => {
  bar();
};
const foo = () => bar();
```

What does `baz()` return?  `baz()` returns `undefined`.  

What does `foo()` return?  *Neither one of us knows* what `foo()` returns, because we don't know what `bar()` is, or what it returns.

Since I am not intending to return a value [here](https://github.com/OpenCollective/opencollective-api/pull/404#discussion-diff-65847947R47), I explicitly use braces.  This will make the code easier to reason about for future developers.  Sure, `console.log()` doesn't return anything, but keeping the braces means a developer doesn't have to worry about it.

So, in fact, I'd recommend others follow this pattern:  if you don't intend to return a value from a lambda, *use braces*.